### DISCUSSION TITLE
2.x版本应该如何实现不可编辑功能呢

### DISCUSSION THREAD

**Discussion by CreateClear** [antvis/X6#3699] [Q&A]
我使用的是2.2.0版本
有些用户仅能查看流程不可编辑 但是2.x版本删除了frozen相关api 查看功能应该如何实现呢

**Comment by wenhaoxu1** [comment]
![image](https://github.com/antvis/X6/assets/107562946/189b197f-d3d9-4ea5-acb6-48e75f07ab19)
